solc/CommandLineInterface.cpp

```diff
@@ -915,20 +915,6 @@ void CommandLineInterface::compile()
 		);
 		solThrow(CommandLineExecutionError, "");
 	}
-	catch (Error const& _error)
-	{
-		if (_error.type() == Error::Type::DocstringParsingError)
-		{
-			report(Error::Severity::Error, *boost::get_error_info<errinfo_comment>(_error));
-			solThrow(CommandLineExecutionError, "Documentation parsing failed.");
-		}
-		else
-		{
-			m_hasOutput = true;
-			formatter.printErrorInformation(_error);
-			solThrow(CommandLineExecutionError, "");
-		}
-	}
```

Comment:
I always thought that we do this sometimes with analysis or parsing errors but turns out we don't. Removing this handler does not result in any failures in the test suite so no properly reported errors need it.

In particular `DocstringParsingErrors` are not fatal so `ErrorReporter::docstringParsingError()` does not throw them. And even if they were, `ErrorReporter` throws `FatalError`, not `util::Error` and that would be caught in `CompilerStack::analyze()`.

I think that we only ever throw `util::Error` at the codegen and later stages, so handling it only in `CompilerStack::compile()` seems enough. It should never get to this handler or the one in `StandardCompiler`.